python/pylibvw.cc

```diff
@@ -37,6 +43,10 @@ typedef boost::shared_ptr<vw> vw_ptr;
 typedef boost::shared_ptr<example> example_ptr;
 typedef boost::shared_ptr<Search::search> search_ptr;
 typedef boost::shared_ptr<Search::predictor> predictor_ptr;
+typedef std::vector<example_ptr> ExList;
```

Comment:
Naming does not match style guidelines. Suggestion: `example_list_t`?

Reply:
renamed